eclipse-scout-core/src/tabbox/SimpleTab.less

```diff
@@ -81,6 +81,43 @@
       &.closer {
         margin-left: 7px;
       }
+
+      &.icon {
```

Comment:
are the rules for icon and message really necessary for the notification badge? margin-right now adds an unnecessary margin.

Reply:
Done 👍🏼